src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;
+
+    public Square(int firstSide, Color color) {
+        this.firstSide = firstSide;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * firstSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(

```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. For example, calculate the area in a separate variable and then use it in the output string.